terraform/application/config/migration.tfvars.json

```diff
@@ -5,5 +5,7 @@
     "deploy_redis_cache": true,
     "redis_cache_family": "C",
     "redis_cache_capacity": 0,
-    "redis_cache_sku_name": "Basic"
+    "redis_cache_sku_name": "Basic",
+    "worker_replicas": 2,
+    "webapp_memory_max": "2Gi"
```

Comment:
maybe we need to bump up the worker instead as that's where the jobs are picked up? Also was the memory an issue or the CPU? I think it was CPU